apps/demos/Demos/Chat/Overview/Angular/app/app.component.css

```diff
@@ -0,0 +1,20 @@
+.demo-container {
+    height: 100%;
```

Comment:
Seems, that this property is excess. Can we remove it?

Reply:
57acc8aa4c51d5f8d5e5517fb13187024a0be1cd